src/Model/Post/UserNotification.php

```diff
@@ -236,6 +266,32 @@ private static function setNotificationForUser(array $item, int $uid)
 		self::update($item['uri-id'], $uid, $fields, true);
 	}
 
+	private static function insertNotication(int $type, int $uid, array $item)
+	{
+		if (($item['gravity'] == GRAVITY_ACTIVITY) &&
+			!in_array($type, [self::NOTIF_DIRECT_COMMENT, self::NOTIF_DIRECT_THREAD_COMMENT])) {
+			// Activities are only stored when performed on the user's post or comment
+			return;
+		}
+
+		$fields = [
+			'uid' => $uid, 
+			'vid' => $item['vid'], 
+			'type' => $type,
+			'actor-id' => $item['author-id'], 
+			'parent-uri-id' => $item['parent-uri-id'],
+			'created' => DateTimeFormat::utcNow(),
+		];
+
+		if ($item['gravity'] == GRAVITY_ACTIVITY) {
+			$fields['target-uri-id'] = $item['thr-parent-id'];
+		} else {
+			$fields['target-uri-id'] = $item['uri-id'];
+		}
+
+		dba::insert('notification', $fields);
```

Comment:
Please use the correct capitalization.
```suggestion
		DBA::insert('notification', $fields);
```

Reply:
Done